### DISCUSSION TITLE
higress如何获取 $remote_addr并放入并覆盖X-Forwarded-For

### DISCUSSION THREAD

**Discussion by zmoon460** [higress-group/higress#1531] [General]
使用nginx 控制器的ingress规则可以设置这样的注解，但在higress中没有效果，需如何实现？

    nginx.ingress.kubernetes.io/configuration-snippet: |
      proxy_set_header X-Forwarded-For $remote_addr;

**Comment by johnlanni** [comment]
higress 默认就会加上这个，不需要单独的注解

**Reply by ishells** [reply]
如果higress外层有ddos高仿之类的代理，同样场景nginx ingress controller需要添加`proxy-real-ip-cidr: x.x.x.x/x`设置高防的回源IP，higress需要添加什么特定的配置来获取真实的客户端IP吗？
